docs/Draft-Generation/using-quote-styles.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Using Quote Styles
+sidebar_position: 3
+slug: /Using-Quote-Styles
+---
+
+# Using Quote Styles
+
+The Quote Style feature automatically restores quotation marks in new drafts so they match the conventions already used in your project. When Scripture Forge generates AI drafts, some punctuation distinctions are lost, this is part of the process and it generally improves the quality of the generated drafts. However, it can create a lot of work for translators who need to restore the distinctions that have been lost. One key distinction is that curly quotation marks are simplified to straight quotes:
+
+'...' (single quotes)
+
+"..." (double quotes)
+
+However, many projects use other styles for quotation marks, such as:
+
+‘ … ’ or “ … ”
+« … »
+「 … 」
+
+On the Generate Draft page is Formatting Options button which will allow you to set up the paragraph breaks and quote styles for your draft. After the initial set up, you can return to the Formatting Options page and change your settings.
+
+![](./paragraph_breaks_image1.png)
+
+You can return to the “Formatting Options” page and change your settings.
+
+![](./paragraph_breaks_image2.png)
+
+## How to Use Quote Styles (step-by-step)
+
+1. Navigate to the Generate Draft page in Scripture Forge.
+2. Select the book you want to draft (e.g. 1 Kings).
+3. Before generating  a new draft, select Formatting Options.
+4. Select the options for quote styles to apply to the draft. The two options are  "Automatic quote style" or "Straight quotes".
+
+![](./quote_styles_image1.png)
+
+5. Click save.
+6. Review the output:
+    If quote styles  are applied, you will see the project’s preferred quotation marks.
+    If the style could not be determined, you’ll receive a clear notification and will need to adjust manually.

```

Comment:
> and will need to adjust manually

I think this could be clearer. This could be understood to mean "you will need to change the setting back to 'straight quotes'", rather than the intended meaning of "you will need to edit the quotes manually when editing the draft"